starlette_prometheus/middleware.py

```diff
@@ -49,3 +50,11 @@ class PrometheusMiddleware(BaseHTTPMiddleware):
             REQUESTS_IN_PROGRESS.labels(method=method, path=path).dec()
 
         return response
+
+    @staticmethod
+    def get_path(request: Request):
```

Comment:
It could be nice to have the signature of the response as well as parameters: `def ... -> str`.

Reply:
Done